vtp-pensjon-application/pom.xml

```diff
@@ -955,6 +974,7 @@
                 <groupId>org.springframework.boot</groupId>
                 <artifactId>spring-boot-maven-plugin</artifactId>
                 <configuration>
+                    <classifier>exec</classifier>
```

Comment:
Trenger å bruke classifier siden denne dras inn i client for test. Har oppdatert Dockerfile for å få med jar fil med exec i navnet